packages/core/useClipboardItems/index.md

```diff
@@ -20,7 +20,7 @@ import { useClipboardItems } from '@vueuse/core'
 const mime = 'text/html'
 const source = ref([
   new ClipboardItem({
-    [mime]: new Blob(['\'<b>HTML content</b>\'', { type: mime }]),
+    [mime]: new Blob([`<b>HTML content</b>`], { type: mime }),
```

Comment:
Thank you for submitting the code!

Since we're not using string interpolation, single quotes would be sufficient.
Please review.

```suggestion
    [mime]: new Blob(['<b>HTML content</b>'], { type: mime }),
```